README.md

```diff
@@ -65,7 +65,7 @@ The key elements of the Context Support are:
 that allows different sub-contexts to be propagated.
 * Languages that already have this support, like [Go][goContext] or C# (ExecutionContext), MUST 
 use the language supported generic context instead of building their own.
-* For an explicit generic context implementation you can look at the Java [io.grpc.Context][gRPCContext].
+* For an implicit generic context implementation you can look at the Java [io.grpc.Context][gRPCContext].
```

Comment:
What does implicit mean in this context. I used to think io.grpc.Context is explicit because the user creates and uses it to explicitly scope a block run in that specific context. Maybe it is too early here to mention implicit/explicit.

Reply:
Yea, this needs more explanation then. I read it and having no knowledge of io.grpc.Context I figured since it was implicit this meant it was some variable automatically tied to a thread by the framework/library.